components/infobox/commons/infobox_basic.lua

```diff
@@ -1,7 +1,10 @@
 local Class = require('Module:Class')
 local Infobox = require('Module:Infobox')
+local String = require('Module:StringUtils')
```

Comment:
```suggestion
local String = require('Module:String')
```

Reply:
Module String is used for stuff directly via invoke (it existed before we started to split the LuaUtils into several modules)
hence we named the string module for usage inside lua `StringUtils`
see https://discord.com/channels/93055209017729024/756217678493909143/852199709669195816

Reply:
I know, it will do what you're looking for 
![image](https://user-images.githubusercontent.com/5138348/130420801-29a27e69-c7a0-458b-bca0-5c4a1b6f8f96.png)